cloudbuild.yaml

```diff
@@ -0,0 +1,20 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 3600s
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20190906-745fed4'
+    entrypoint: bash
+    env:
+    - PROW_GIT_TAG=$_GIT_TAG
+    - PULL_BASE_REF=$_PULL_BASE_REF
+    args:
+    - -c
+    - |
+      echo "Building/Pushing DNS containers"
+      make all-push
+substitutions:
+  # _GIT_TAG will be filled with a git-based tag for the image, of the form vYYYYMMDD-hash, and
+  # can be used as a substitution
+  _GIT_TAG: '12345'
```

Comment:
Do we need to modify this when releasing or will it be automatically filled?

Reply:
I think we don't need to populate this, when we call make all-push after the tag, it works fine. I need to trigger the whole flow once to be sure though. I was not able to run this by hand due to permission issues. So I have added the prow yaml in https://github.com/kubernetes/test-infra/pull/17123 which is the recommended way.